orchagent/pfcwdorch.cpp

```diff
@@ -447,6 +451,14 @@ void PfcWdSwOrch<DropHandler, ForwardHandler>::unregisterFromWdDb(const Port& po
         // Unregister in syncd
         m_pfcWdTable->del(key);
         m_entryMap.erase(queueId);
+
+        // Clean up
+        RedisClient redisClient(PfcWdOrch<DropHandler, ForwardHandler>::getCountersDb().get());
+        string countersKey = COUNTERS_TABLE ":" + sai_serialize_object_id(queueId);
+        redisClient.hdel(countersKey, "PFC_WD_DETECTION_TIME");
```

Comment:
Can we use macro PFC_WD_RESTORATION_TIME etc here? Also what about using for loop to remove all needed fields?

Reply:
@sihuihan88 , maybe I am missing something, but the macro PFC_WD_DETECTION_TIME is different from "PFC_WD_DETECTION_TIME". From the code, I cannot find PFC_WD_DETECTION_TIME used anywhere.